core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -256,19 +251,41 @@ class ZkUtils(val zkClient: ZkClient,
   }
 
   /**
-   * Register brokers with v2 json format (which includes multiple endpoints).
+   * Register brokers with v3 json format (which includes multiple endpoints and rack).
```

Comment:
We say that we register the v3 format including rack, but the code only adds the rack if `apiVersion >= 3`. If we follow the same approach as what we did with V2, we should always add the rack as it will simply be ignored if `version == 2`, right?